client/src/App.js

```diff
@@ -362,6 +385,13 @@ function App () {
     }
   };
 
+  const renderAppSounds = () => {
+    const sounds = Object.values(GameStateConstants.GAME_SOUNDS);
+    return sounds.map(sound => (
+      <Audio name={sound} key={sound} />
```

Comment:
@existentialcoder : 
Instead of creating multiple Audio components in the dom, can we have 
- A single Audio component
- Control which audio needs to be played with the help of props passed into the component
- Inside the Audio component, set the audio source based on the passed in property
- We can also try moving the play call within the Audio component. 
- That way, all the App component does is to change the property for the Audio component whenever an audio needs to be played. 

Your thoughts on this ?